src/ireiat/data_pipeline/assets/demand/faf5_to_county.py

```diff
@@ -85,7 +85,7 @@ def faf_id_to_county_areas(
     assert min(county_coverage_canonical.values()) == 1.0
 
     # ensure every faf zone is covered
-    assert (set(faf5_regions_src["faf_zone"]) - set(faf_zone_to_county_canonical.keys())) == set()
+    assert (set(faf5_regions_src["FAF_Zone"]) - set(faf_zone_to_county_canonical.keys())) == set()
```

Comment:
oh let's be careful about this. Maybe we strictly lowercase all of the incoming string upon ingestion to avoid capitalization errors....

But I'm sure there are some downstream impacts to this :)

Reply:
I'll give this a try to see if it messes with anything downstream